test/Distributed/Runtime/distributed_actor_localSystem_distributedProtocol.swift

```diff
@@ -1,5 +1,6 @@
 // RUN: %empty-directory(%t)
 // RUN: %target-build-swift -module-name main -j2 -parse-as-library -I %t %s -plugin-path %swift-plugin-dir -o %t/a.out
+// RUN: %target-build-swift -module-name main -j2 -parse-as-library -enable-library-evolution -I %t %s -plugin-path %swift-plugin-dir -o %t/a.out
```

Comment:
I think if you want to test both with and without library-evolution you need to duplicate the code-sign + run that lower here as well, because all this did is just rebuilds with library-evolution enabled and and executes that. The same applies to all other modified tests here.

Reply:
Ugh, you're right ofc, thank you for noticing. Yeap, we'll want to build+run in both to make sure. 

Current problem would be crashing just during compilation but we want to verify run as well.